air/src/lib.rs

```diff
@@ -66,11 +66,16 @@ impl Air for ProcessorAir {
         let mut chiplets_degrees = chiplets::get_transition_constraint_degrees();
         main_degrees.append(&mut chiplets_degrees);
 
+        // --- stack constraints -------------------------------------------------------------------
+        let mut stack_degrees = stack::get_transition_constraint_degrees();
+        main_degrees.append(&mut stack_degrees);
+
         // Define the transition constraint ranges.
         let constraint_ranges = TransitionConstraintRange::new(
             1,
             range::get_transition_constraint_count(),
             chiplets::get_transition_constraint_count(),
+            stack::get_transition_constraint_count(),
```

Comment:
nit: I'd probably switch the order to be consistent with the order we usually show on diagrams etc. Specifically, stack is usually listed before range and chiplets components.

Reply:
Updated in the latest PR.